### PR TITLE
Added `rule_id` to `/api/v1/rules/query`.

### LINKED ISSUE #7: Return rule ID in /api/v1/rules/query
Currently ID isn't part of the data returned, which introduces a cumbersome API where to fetch the ID we need to specifically search.

### PR DESCRIPTION
**Pull request by aviramha**
.gitignore fixes

scripts chmodded and shebanged

changelog
Closes #7

### CODE REVIEW

File: heksher/api/v1/rules.py [hunk 1]
```diff
@@ -120,6 +120,7 @@ class QueryRulesOutput_Rule(ORJSONModel):
     context_features: List[Tuple[str, str]] = Field(
         description="a list of exact-match conditions for the rule, in hierarchical order"
     )
+    rule_id: int
```
Comment (bentheiii) [inline heksher/api/v1/rules.py:123]:
```suggestion
    rule_id: int = Field(description=...)
```

**Review by bentheiii — COMMENTED** [review]
Are we sure we always want the rule_id, maybe it should be on metadata only?

File: heksher/api/v1/rules.py (hunk 1 above)
Comment (aviramha) [reply heksher/api/v1/rules.py:123]: The rules are managed using rule IDs, I think it's elementary information.

**Comment by aviramha** [comment]
> Are we sure we always want the rule_id, maybe it should be on metadata only?

The rules are managed using rule IDs, I think it's elementary information.

**Review by bentheiii — APPROVED** [review]

**Pull request merged**